packages/nimble-components/src/combobox/index.ts

```diff
@@ -154,6 +154,19 @@ export class Combobox
         this.filteredOptions = enabledOptions;
     }
 
+    protected override focusAndScrollOptionIntoView(): void {
```

Comment:
> . Seems like having the nimble combobox override the method and adding the same check before calling super would be reasonable there:

```ts
protected override focusAndScrollOptionIntoView(): void {
  if(this.open) {
    super.focusAndScrollOptionIntoView()
  }
}
```
unless there is an important reason to duplicate the code slightly differently...